Frontend/package.json

```diff
@@ -43,5 +44,9 @@
       "json",
       "node"
     ]
+  },
+  "dependencies": {
+    "koa": "^2.4.1",
```

Comment:
Why `Koa`?

Reply:
Koa lends itself better to a true middleware approach, which will be invaluable when we add the react middleware with auth, etc. What were you thinking instead? `express`? I'd be open to that, but I prefer `koa`, personally.